dali/python/nvidia/dali/ops/__init__.py

```diff
@@ -206,75 +311,67 @@ def _add_spec_args(schema, spec, kwargs):
 
 
 class _OperatorInstance(object):
+    """Class representing the operator node in DALI Pipeline/graph.
+    The entry point accepts grouped arguments: inputs (positional arguments with DataNodes),
+    arguments (dictionary with scalar values), and argument inputs (dictionary with DataNodes).
+
+    It is responsible for the full* validation of the three input/argument kinds and their
+    additional processing (scalar promotions, deprecations and renaming, splitting for conditionals)
+
+    Each "_process_[arguments/inputs]" step adds them to the OpSpec specifying the operator for
+    the C++ backend.
 
-    def __init__(self, inputs, op, **kwargs):
+    Parameters
+    ----------
+    object : _type_
+        _description_
+    """
+
+    def __init__(self, inputs, arguments, arg_inputs, op):
```

Comment:
Consider propagating the arguments, that were already added to the spec in __init__ for full visibility.